docs/reference/api/widgets/button.rst

```diff
@@ -29,14 +29,8 @@ A button has a text label. A handler can be associated with button press events.
 
     button = toga.Button('Click me', on_press=my_callback)
 
-Notes
```

Comment:
I think we should keep this as a "Notes" section. While the notes at present are all style related, we're very likely going to come across platform specific eccentricities that don't fit the description being "style", and don't map to a specific attribute like the text handling of `None` et al.